### PR TITLE
Add NoelShack.com.xml

### LINKED ISSUE #18097: New ruleset: noelshack.com
Type: new ruleset

Domain: noelshack.com

Got this via email:

> Hello to all the team!
>
> I'm contact you to report a problem with the "HTTPS Everywhere" extension. 
> Indeed, when I access the "NoelShack" website by typing www.noelshack.com, 
> the extension is supposed to redirect me to https://www.noelshack.com/ but 
> this is not the case. So I don't see any use for this extension.... Can you
> fix this problem?

### PR DESCRIPTION
**Pull request by pipboy96**
Closes #18097.

### CODE REVIEW

**Comment by Hainish** [comment]
Thanks @pipboy96!

File: src/chrome/content/rules/NoelShack.com.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--
+	Connection closed:
+		- dev
```
Comment (J0WI) [inline src/chrome/content/rules/NoelShack.com.xml]: Wildcard DNS record

Comment (pipboy96) [reply src/chrome/content/rules/NoelShack.com.xml]: Done.

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**